addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,14 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
+        <div t-attf-class="o_notification toast show border-{{props.type}} overflow-hidden mb-2"
         role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
-                aria-label="Close" t-on-click="props.close">
-                    <span class="d-inline" aria-hidden="true">×</span>
+            <div t-attf-class="toast-header align-items-start justify-content-end pb-0 ">
+                <strong t-if="props.title" t-attf-class="o_notification_title text-{{props.type}} mr-auto" t-esc="props.title"/>
+                <button type="button" class="o_notification_close close ml-2" aria-label="Close" t-on-click="props.close">
+                    <span class="d-inline-block" aria-hidden="true">×</span>
                 </button>
-                <div t-if="props.title" class="font-weight-bold o_notification_title mb-1" t-esc="props.title"/>
+            </div>
+            <div t-attf-class="o_notification_body toast-body mr-4 px-3 py-2">
```

Comment:
Is the `mr-4` still useful?

Reply:
Removing it. It was more a design choice (that no text goes under close btn area), but it's actually fine that it does.